pyth-rng/src/config/register_provider.rs

```diff
@@ -0,0 +1,23 @@
+use {
+    crate::config::{
+        EthereumOptions,
+        RandomnessOptions,
+    },
+    clap::Args,
+};
+
+#[derive(Args, Clone, Debug)]
+#[command(next_help_heading = "Register Provider Options")]
+#[group(id = "RegisterProvider")]
+pub struct RegisterProviderOptions {
+    #[command(flatten)]
+    pub ethereum: EthereumOptions,
+
+    #[command(flatten)]
+    pub randomness: RandomnessOptions,
+
+    /// The fee to charge (in wei) for each requested random number
+    #[arg(long = "pyth-contract-fee")]
+    #[arg(default_value = "100")]
+    pub fee: u64,
```

Comment:
`u64` can be low in some chains (where the native token doesn't have much value)